lib/media/transmuxer.js

```diff
@@ -97,6 +115,32 @@ shaka.media.Transmuxer = class {
   }
 
 
+  /**
+   * For any stream, convert its codecs to MP4 codecs.
+   * @param {string} contentType
+   * @param {string} mimeType
+   * @return {string}
+   */
+  static convertCodecs(contentType, mimeType) {
+    const Transmuxer = shaka.media.Transmuxer;
+    if (Transmuxer.isAacContainer(mimeType)) {
+      return Transmuxer.convertAacCodecs();
+    } else if (Transmuxer.isTsContainer(mimeType)) {
+      return Transmuxer.convertTsCodecs(contentType, mimeType);
+    }
+    return mimeType;
+  }
+
+
+  /**
+   * For aac stream, convert its codecs to MP4 codecs.
+   * @return {string}
+   */
+  static convertAacCodecs() {
```

Comment:
If we're sending all access through convertCodecs(), please make convertAacCodecs() and convertTsCodecs() private to catch any direct usage of them we missed.  (Unless there's some reason we should keep these public.)

Reply:
Done!